.goreleaser.yml

```diff
@@ -108,6 +108,12 @@ archives:
       - src: release_deps/observiq-otel-collector.service
         dst: "install"
         strip_parent: true
+      - src: release_deps/observiq-otel-collector
+        dst: "install"
+        strip_parent: true
+      - src: release_deps/sysconfig/observiq-otel-collector
+        dst: "install/sysconfig"
+        strip_parent: true
```

Comment:
This is what it looks like when extracted. Kind of confusing. (two `observiq-otel-collector`)

```
install
├── com.observiq.collector.plist
├── observiq-otel-collector
├── observiq-otel-collector.service
├── sysconfig
│   └── observiq-otel-collector
└── windows_service.json
```

I would be in favor of removing service files from archives, but thats Brandon / Corbin's call.

Reply:
I feel they need to be in the archive for pure manual installs. I know these are rare, but they do happen fro time to time.